app/src/pages/ExploreDataPage.tsx

```diff
@@ -6,18 +6,49 @@ import FormControl from "@material-ui/core/FormControl";
 import MenuItem from "@material-ui/core/MenuItem";
 import DemoReport from "../features/reports/DemoReport";
 import TellMeAboutReport from "../features/reports/TellMeAboutReport";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import IconButton from "@material-ui/core/IconButton";
+import ShareIcon from "@material-ui/icons/Share";
+import { getMadLibPhraseText } from "../utils/MadLibs";
+
 import {
   MADLIB_LIST,
   MadLib,
   PhraseSegment,
   PhraseSelections,
 } from "../utils/MadLibs";
 import styles from "./ExploreDataPage.module.scss";
+import {
+  clearSearchParams,
+  buildMadLibSelectionParams,
+  MADLIB_PHRASE,
+  MADLIB_SELECTIONS,
+  useSearchParams,
+} from "../utils/urlutils";
 
 function ExploreDataPage() {
-  const [phraseIndex, setPhraseIndex] = useState(0);
+  const [shareModalOpen, setShareModalOpen] = React.useState(false);
+  const params = useSearchParams();
+  useEffect(() => {
+    clearSearchParams([MADLIB_PHRASE, MADLIB_SELECTIONS]);
```

Comment:
I wonder if rather than clear the params, we should keep them updated in the url. That way if you reload the page you'll go back to the same report you were on. Also so you have the option of sharing just by copy/pasting the main url. I realize that this may be more work though, so feel free to skip, though I'm curious if you agree in general

Reply:
Oh also we can probably get back navigation working if we keep the url params

Reply:
Actually my instinct was the same but then we have to update them programatically. I'll add as a TODO for now since this is lower pri